src/LibraryManager/Json/LibraryInstallationStateOnDisk.cs

```diff
@@ -10,12 +10,12 @@ namespace Microsoft.Web.LibraryManager.Json
 {
     internal class LibraryInstallationStateOnDisk
     {
-        [JsonProperty(ManifestConstants.Library)]
```

Comment:
The re-ordering here is so that the serialized object matches the order that one would type in the editor (provider has to be first in order to get completion for remaining properties).  This also matches the screenshots we have in the wiki documentation.